### PR TITLE
Updated angular to 1.5.0-rc.0 on 03-ui-router-basic-states

### PR DESCRIPTION
**Pull request by Creeland**
@ijones16

### CODE REVIEW

**Comment by ijones16** [comment]
@Creeland This one isn't loading, There aren't any errors though. Also, could you bump the ui-router version up?

**Comment by Creeland** [comment]
@ijones16 None of the previous commits load either. So my updating of Angular to 1.5.0-rc.0 isn't the issue.

File: src/app/eggly-app.js [hunk 1]
```diff
@@ -32,41 +34,41 @@ angular.module('Eggly', [
...
+        main.setEditedBookmark = setEditedBookmark;
+        main.isSelectedBookmark = isSelectedBookmark;
 
         function resetCreateForm() {
-            $scope.newBookmark = {
+            main.newBookmark = {
                 title: '',
                 url: '',
-                category: $scope.currentCategory
+                category: main.currentCategory
```
Comment (ijones16) [inline src/app/eggly-app.js]: this needs to be main.currentCategory.name

**Comment by ijones16** [comment]
this branch has been updated

**Pull request closed without merging**